pkg/transforms/common.go

```diff
@@ -548,9 +548,14 @@ func edgesToApplication(nodeInfo NodeInfo, ns NodeStore, UID string, onlyApplica
 }
 
 type relatedObject struct {
-	kind      string
-	namespace string
-	name      string
+	apiVersion string
```

Comment:
Should also add `apiGroup` here.

Reply:
This `apiVersion` is both, like in a kubernetes object. Is there a better term for just the `v1` part?

Reply:
You are right, it's ok to keep it combined here. An alternative is to name it `apiGroupVersion`.  If you keep the current name please add a comment like `// format:  apigroup/version` so I remember in the future.

The reason it's slightly confusing is because we divided it so it's easier to search by `apiGroup` or `apiVersion`. [Reference code here.](https://github.com/stolostron/search-collector/blob/main/pkg/transforms/common.go#L596-L605)